cds_migrator_kit/rdm/migration/README.md

```diff
@@ -299,3 +299,48 @@ reindex_stats(stats_indices)
 ```
 
 visit https://migration-cds-rdm-dev.app.cern.ch for report
+
+## Rerun migration from clean state without setup everything again
+
+If you want to cleanup a previous migration run without having to re setup everything
+i.e not repopulating all vocabularies which takes a lot of time, then run the following
+recipe:
+
+- Cleanup db tables from pgadmin
+
+```sql
+DELETE FROM rdm_versions_state;
+DELETE FROM rdm_records_files;
+DELETE FROM rdm_drafts_files;
+DELETE FROM rdm_records_metadata;
+DELETE FROM rdm_drafts_metadata;
+DELETE FROM rdm_parents_metadata;
+DELETE FROM communities_metadata;
```

Comment:
no need to remove the community, if you omit this we avoid recreating it later

Reply:
we cannot do that because of files cleanup.... there is an fk to community metadata